kolibri/core/auth/models.py

```diff
@@ -1406,3 +1414,52 @@ def remove_learner(self, user):
 
     def __str__(self):
         return self.name
+
+
+@python_2_unicode_compatible
+class IndividualLearnersGroup(Collection):
+    """
+    An ``IndividualLearnersGroup`` is a collection kind that can be used in an assignment
+    to create a group that is specific to a single ``Lesson`` or ``Exam``.
+    """
```

Comment:
as far as naming ... `IndividualLearnersGroup` and `LearnerGroup` are pretty similar.

Maybe a name like `LearnerSelection` or `AdHocGroup`? These imply to me that it's a one-off creation intended for things like assignments...

Reply:
Renamed throughout to AdHocGroup.

The only thing that didn't change is the `IndividualLearnerSelecter.vue` file because strings are associated with it.